scout/commands/update/individual.py

```diff
@@ -1,22 +1,34 @@
-"""Code for updating information on individuals"""
+"""Code for updating information on individuals
+"""
 from pathlib import Path
 
 import click
 
 from scout.server.extensions import store
 
-UPDATE_KEYS = [
-    "bam_file",
-    "mt_bam",
-    "vcf2cytosure",
-    "rhocall_bed",
-    "rhocall_wig",
-    "tiddit_coverage_wig",
-    "upd_regions_bed",
-    "upd_sites_bed",
-    "splice_junctions_bed",  # An indexed junctions .bed.gz file obtained from STAR v2 aligner *.SJ.out.tab file.
-    "rna_coverage_bigwig",  # Coverage islands generated from bam or cram files (RNA-seq analysis)
-]
+UPDATE_DICT = {
```

Comment:
Great to make some order in these fields! 

I like also a boolean solution:

UPDATE_FIELD_IS_PATH = {
  "bam_file": True,
  "bionano_access.sample": False,
   ....
}

Reply:
It would, and with less memory; I was thinking we might add more "types"..